tests/unit/NotifyQueueConsumerTest/Queue/ConsumerTest.php

```diff
@@ -93,6 +96,63 @@ public function testFetchMessagePostLetterSendToNotifyUpdateStatusFailsOnce(): v
         $this->assertTrue($this->sleepMockCalled);
     }
 
+    /**
+     * @throws Exception
+     */
+    public function testFetchMessagePostLetterSendToNotifyUpdateStatusFailsWithClientException(): void
+    {
+        $command = $this->createSendToNotifyCommand();
+        $updateDocumentStatusCommand = $this->createUpdateDocumentStatusCommand();
+
+        $this->queueMock->expects(self::once())->method('next')->willReturn($command);
+        $this->sendToNotifyHandlerMock
+            ->expects(self::once())
+            ->method('handle')
+            ->with($command)
+            ->willReturn($updateDocumentStatusCommand);
+        $this->queueMock->expects(self::once())->method('delete')->with($command);
+        $this->updateDocumentStatusHandlerMock
+            ->expects(self::exactly(2))
+            ->method('handle')
+            ->withConsecutive([$updateDocumentStatusCommand], [$updateDocumentStatusCommand])
+            ->will($this->onConsecutiveCalls($this->throwException(
+                new ClientException(
+                    'some message',
+                    new Request('put', '/'),
+                    new Response(404, [], 'this is the problem')
+                )
+            ), $this->returnValue(null)));
+        $this->loggerMock->expects(self::never())->method('critical');
+
+        $this->consumer->run();
+
+        $this->assertTrue($this->sleepMockCalled);
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function testFetchMessageEmailInvoiceSendToNotifyUpdateStatusSuccess(): void
```

Comment:
Is this test not identical to `testFetchMessagePostLetterSendToNotifyUpdateStatusSuccess` above? (since "email invoice" doesn't do anything special in `createSendToNotifyCommand`)

Reply:
Whoops, thats come from a merge conflict, my bad! Will fix that now